pems/README.md

```diff
@@ -12,7 +12,9 @@ In this location there are only three lanes so only three sensors of information
 
 ## Table of contents
 
-- `parquet`: directory containing partitioned Parquet data
+- `parquet`: Directory containing partitioned Parquet data.
+- `_common_metadata`: File contains the schema for the whole dataset, including those hierarchical columns and those not in this specific section of the data.
+- `_metadata`: File contains the schema used for part files (omitting the hierarchical columns) and per-file column stats (min, max, etc.,) for all the files, with their complete relative path names.
```

Comment:
These descriptions contain lingo that someone that knows the details of what is going on would understand, but I'm left trying to guess.  Can we rephrase these to hit readers that aren't as educated on what is going on?